docs/project/breaking-change-process.md

```diff
@@ -0,0 +1,44 @@
+# Breaking change process
+
+Breaking changes make it difficult for people to adopt new versions of .NET. We should avoid making them. In the rare case that you believe a breaking change is unavoidable, you should follow the process outlined in this document. The intent of the process is to get feedback. This might result in previously unseen  opportunities to achieve your goals without breaks, clever ways to limit breaks to more niche scenarios, and (most importantly) better understand the scope of impact to .NET users.
+
+This process does not result in approval or denial of a breaking change request. It results in feedback, which might be varied or uniform. In the best case, the feedback is a call to action to you, that provides a direction on how you should proceed. The people that are responsible for reviewing your PR are expected to use the information collected by this process as part of their determination of whether your change is appropriate to merge.
+
+You should start this process as soon as possible, preferably before any code is written, in the design phase. The breaking change issue should be paired with a design document that helps reviewers understand the broader scope, goals and constraints of your change. If you don't yet have a design document, you are encouraged to first write such a document, to enable people giving feedback to develop a deeper level of understanding on your approach. If you do not do this, you might stall the process.
+
+There are people on the team that can help you work through a breaking change, if you want to engage with them before starting this more formal process. They are part of the [dotnet/compat team](https://github.com/orgs/dotnet/teams/compat) . Feel free to @mention them on issues to start a dialogue.
+
+## Process
```

Comment:
Are we supposed to create new issues for issues that track design work in case this design has been identified as potentially being breaking or is it OK if we apply this label to such issues?

Reply:
That one is easy. You should create an issue when you start wanting feedback (not solely on compatibility).